extensions/blocks/contact-form/index.js

```diff
@@ -47,7 +47,7 @@ export const settings = {
 	},
 	attributes: defaultAttributes,
 	edit,
-	save: InnerBlocks.Content,
+	save,
```

Comment:
Don't we need a new deprecation to handle the new save function on existing blocks?